kuka_external_control_sdk/changelog

```diff
@@ -1,3 +1,10 @@
+kuka-external-control-sdk (1.2.1)
+
+  * Add message builder tests
+  * Make ipoc unsigned
+
+ -- Team Apollo <KUKA-HU-RD-TeamApollo@kuka.com>  Tue, 02 Apr 2024 13:12:00 +0000
```

Comment:
Should we present the changelog here? And if so, is it okay to include a company email address even though changes could be accepted by developers outside KUKA?

Reply:
good point, we should change to a different changelog format